### PR TITLE
bpo-43269: Clean up sqlite3 file scope

### PR DESCRIPTION
**Pull request by erlend-aasland**
https://bugs.python.org/issue43269

### CODE REVIEW

**Comment by erlend-aasland** [comment]
_skip news_

File: Modules/_sqlite/cache.c [hunk 1]
```diff
@@ -113,7 +117,8 @@ void pysqlite_cache_dealloc(pysqlite_Cache* self)
     Py_DECREF(tp);
 }
 
-PyObject* pysqlite_cache_get(pysqlite_Cache* self, PyObject* key)
+extern PyObject *
```
Comment (serhiy-storchaka) [inline Modules/_sqlite/cache.c]: Is not `extern` redundant?

**Review by serhiy-storchaka — COMMENTED** [review]
Are there redundant declarations in header files?

File: Modules/_sqlite/cache.c (hunk 1 above)
Comment (erlend-aasland) [reply Modules/_sqlite/cache.c]: Yes, but I like the explicity. I'm happy to remove it, though. Let me know what you prefer.

**Comment by erlend-aasland** [comment]
> Are there redundant declarations in header files?

I already clean up those in a previous PR/issue. I'd like to get rid of the forward declarations in the top of the c files though, but that'll require moving a couple of the functions around.

File: Modules/_sqlite/cache.c (hunk 1 above)
Comment (serhiy-storchaka) [reply Modules/_sqlite/cache.c]: Explicit `extern` is not commonly use in CPython sources, so it would be better to remove it.

Comment (erlend-aasland) [reply Modules/_sqlite/cache.c]: All right, I'll do that. Thanks.

Comment (erlend-aasland) [reply Modules/_sqlite/cache.c]: Fixed in bd5c34559f7b85328e0f90a67888d8dc1e740b1a. PTAL.

File: Modules/_sqlite/cache.c [hunk 2]
```diff
@@ -113,7 +117,8 @@ void pysqlite_cache_dealloc(pysqlite_Cache* self)
     Py_DECREF(tp);
 }
 
-PyObject* pysqlite_cache_get(pysqlite_Cache* self, PyObject* key)
+PyObject *
+pysqlite_cache_get(pysqlite_Cache *self, PyObject *key)
```
Comment (berkerpeksag) [inline Modules/_sqlite/cache.c]: I really don't like these unrelated cosmetic changes. The scope of this PR is adding missing `static` keywords and it's OK to follow PEP 7 while adding them. However, changes like these are out of scope of this PR.

File: Modules/_sqlite/cache.c [hunk 3]
```diff
@@ -291,7 +297,8 @@ static PyType_Spec cache_spec = {
 };
 PyTypeObject *pysqlite_CacheType = NULL;
 
-extern int pysqlite_cache_setup_types(PyObject *mod)
+int
+pysqlite_cache_setup_types(PyObject *mod)
```
Comment (berkerpeksag) [inline Modules/_sqlite/cache.c]: It would be great if you could submit a separate PR for removing redundant `extern`s.

File: Modules/_sqlite/connection.c [hunk 4]
```diff
@@ -255,7 +259,9 @@ void pysqlite_connection_dealloc(pysqlite_Connection* self)
  *
  * 0 => error; 1 => ok
  */
-int pysqlite_connection_register_cursor(pysqlite_Connection* connection, PyObject* cursor)
+int
```
Comment (berkerpeksag) [inline Modules/_sqlite/connection.c]: Same comment about unrelated cosmetic changes.

File: Modules/_sqlite/connection.c [hunk 5]
```diff
@@ -356,7 +362,8 @@ pysqlite_connection_close_impl(pysqlite_Connection *self)
  *
  * 0 => error; 1 => ok
  */
-int pysqlite_check_connection(pysqlite_Connection* con)
+int
```
Comment (berkerpeksag) [inline Modules/_sqlite/connection.c]: Same comment about unrelated cosmetic changes.

File: Modules/_sqlite/connection.c [hunk 6]
```diff
@@ -371,7 +378,8 @@ int pysqlite_check_connection(pysqlite_Connection* con)
     }
 }
 
-PyObject* _pysqlite_connection_begin(pysqlite_Connection* self)
+PyObject *
```
Comment (berkerpeksag) [inline Modules/_sqlite/connection.c]: Same comment about unrelated cosmetic changes.

File: Modules/_sqlite/connection.c [hunk 7]
```diff
@@ -509,7 +517,7 @@ pysqlite_connection_rollback_impl(pysqlite_Connection *self)
 }
 
 static int
-_pysqlite_set_result(sqlite3_context* context, PyObject* py_val)
+_pysqlite_set_result(sqlite3_context *context, PyObject *py_val)
```
Comment (berkerpeksag) [inline Modules/_sqlite/connection.c]: Same comment about unrelated cosmetic changes.

File: Modules/_sqlite/connection.c [hunk 8]
```diff
@@ -601,7 +611,8 @@ PyObject* _pysqlite_build_py_params(sqlite3_context *context, int argc, sqlite3_
 }
 
 static void
-_pysqlite_func_callback(sqlite3_context *context, int argc, sqlite3_value **argv)
+_pysqlite_func_callback(sqlite3_context *context, int argc,
```
Comment (berkerpeksag) [inline Modules/_sqlite/connection.c]: Same comment about unrelated cosmetic changes.

File: Modules/_sqlite/connection.c [hunk 9]
```diff
@@ -637,7 +648,9 @@ _pysqlite_func_callback(sqlite3_context *context, int argc, sqlite3_value **argv
     PyGILState_Release(threadstate);
 }
 
-static void _pysqlite_step_callback(sqlite3_context *context, int argc, sqlite3_value** params)
+static void
```
Comment (berkerpeksag) [inline Modules/_sqlite/connection.c]: Same comment about unrelated cosmetic changes.

File: Modules/_sqlite/connection.c [hunk 10]
```diff
@@ -751,7 +764,8 @@ _pysqlite_final_callback(sqlite3_context *context)
     PyGILState_Release(threadstate);
 }
 
-static void _pysqlite_drop_unused_statement_references(pysqlite_Connection* self)
+static void
```
Comment (berkerpeksag) [inline Modules/_sqlite/connection.c]: Same comment about unrelated cosmetic changes.

File: Modules/_sqlite/connection.c [hunk 11]
```diff
@@ -782,7 +796,8 @@ static void _pysqlite_drop_unused_statement_references(pysqlite_Connection* self
     Py_SETREF(self->statements, new_list);
 }
 
-static void _pysqlite_drop_unused_cursor_references(pysqlite_Connection* self)
+static void
```
Comment (berkerpeksag) [inline Modules/_sqlite/connection.c]: Same comment about unrelated cosmetic changes.

File: Modules/_sqlite/connection.c [hunk 12]
```diff
@@ -813,7 +828,8 @@ static void _pysqlite_drop_unused_cursor_references(pysqlite_Connection* self)
     Py_SETREF(self->cursors, new_list);
 }
 
-static void _destructor(void* args)
+static void
```
Comment (berkerpeksag) [inline Modules/_sqlite/connection.c]: Same comment about unrelated cosmetic changes.

File: Modules/_sqlite/connection.c [hunk 13]
```diff
@@ -914,7 +930,10 @@ pysqlite_connection_create_aggregate_impl(pysqlite_Connection *self,
     Py_RETURN_NONE;
 }
 
-static int _authorizer_callback(void* user_arg, int action, const char* arg1, const char* arg2 , const char* dbname, const char* access_attempt_source)
+static int
```
Comment (berkerpeksag) [inline Modules/_sqlite/connection.c]: Same comment about unrelated cosmetic changes.

File: Modules/_sqlite/connection.c [hunk 14]
```diff
@@ -953,7 +972,8 @@ static int _authorizer_callback(void* user_arg, int action, const char* arg1, co
     return rc;
 }
 
-static int _progress_handler(void* user_arg)
+static int
```
Comment (berkerpeksag) [inline Modules/_sqlite/connection.c]: Same comment about unrelated cosmetic changes.

File: Modules/_sqlite/connection.c [hunk 15]
```diff
@@ -987,9 +1007,12 @@ static int _progress_handler(void* user_arg)
  * may change in future releases. Callback implementations should return zero
  * to ensure future compatibility.
  */
-static int _trace_callback(unsigned int type, void* user_arg, void* prepared_statement, void* statement_string)
+static int
```
Comment (berkerpeksag) [inline Modules/_sqlite/connection.c]: Same comment about unrelated cosmetic changes.

File: Modules/_sqlite/connection.c [hunk 16]
```diff
@@ -987,9 +1007,12 @@ static int _progress_handler(void* user_arg)
...
+_trace_callback(unsigned int type, void *user_arg, void *prepared_statement,
+                void *statement_string)
 #else
-static void _trace_callback(void* user_arg, const char* statement_string)
+static void
```
Comment (berkerpeksag) [inline Modules/_sqlite/connection.c]: Same comment about unrelated cosmetic changes.

File: Modules/_sqlite/connection.c [hunk 17]
```diff
@@ -1197,7 +1220,8 @@ pysqlite_connection_load_extension_impl(pysqlite_Connection *self,
 }
 #endif
 
-int pysqlite_check_thread(pysqlite_Connection* self)
+int
```
Comment (berkerpeksag) [inline Modules/_sqlite/connection.c]: Same comment about unrelated cosmetic changes.

File: Modules/_sqlite/connection.c [hunk 18]
```diff
@@ -1212,12 +1236,16 @@ int pysqlite_check_thread(pysqlite_Connection* self)
     return 1;
 }
 
-static PyObject* pysqlite_connection_get_isolation_level(pysqlite_Connection* self, void* unused)
+static PyObject *
```
Comment (berkerpeksag) [inline Modules/_sqlite/connection.c]: Same comment about unrelated cosmetic changes.

File: Modules/_sqlite/connection.c [hunk 19]
```diff
@@ -1212,12 +1236,16 @@ int pysqlite_check_thread(pysqlite_Connection* self)
...
+pysqlite_connection_get_isolation_level(pysqlite_Connection *self,
+                                        void *Py_UNUSED(unused))
 {
     return Py_NewRef(self->isolation_level);
 }
 
-static PyObject* pysqlite_connection_get_total_changes(pysqlite_Connection* self, void* unused)
+static PyObject *
```
Comment (berkerpeksag) [inline Modules/_sqlite/connection.c]: Same comment about unrelated cosmetic changes.

File: Modules/_sqlite/connection.c [hunk 20]
```diff
@@ -1226,7 +1254,9 @@ static PyObject* pysqlite_connection_get_total_changes(pysqlite_Connection* self
     }
 }
 
-static PyObject* pysqlite_connection_get_in_transaction(pysqlite_Connection* self, void* unused)
+static PyObject *
```
Comment (berkerpeksag) [inline Modules/_sqlite/connection.c]: Same comment about unrelated cosmetic changes.

File: Modules/_sqlite/connection.c [hunk 21]
```diff
@@ -1238,7 +1268,9 @@ static PyObject* pysqlite_connection_get_in_transaction(pysqlite_Connection* sel
 }
 
 static int
-pysqlite_connection_set_isolation_level(pysqlite_Connection* self, PyObject* isolation_level, void *Py_UNUSED(ignored))
+pysqlite_connection_set_isolation_level(pysqlite_Connection *self,
```
Comment (berkerpeksag) [inline Modules/_sqlite/connection.c]: Same comment about unrelated cosmetic changes.

File: Modules/_sqlite/connection.c [hunk 22]
```diff
@@ -1457,10 +1491,9 @@ pysqlite_connection_executescript(pysqlite_Connection *self,
 /* ------------------------- COLLATION CODE ------------------------ */
 
 static int
-pysqlite_collation_callback(
-        void* context,
-        int text1_length, const void* text1_data,
-        int text2_length, const void* text2_data)
+pysqlite_collation_callback(void *context, int text1_length,
```
Comment (berkerpeksag) [inline Modules/_sqlite/connection.c]: Same comment about unrelated cosmetic changes.

File: Modules/_sqlite/connection.c [hunk 23]
```diff
@@ -1919,7 +1952,8 @@ static PyType_Spec connection_spec = {
 
 PyTypeObject *pysqlite_ConnectionType = NULL;
 
-extern int pysqlite_connection_setup_types(PyObject *module)
+int
```
Comment (berkerpeksag) [inline Modules/_sqlite/connection.c]: This can be done in a separate PR.

File: Modules/_sqlite/cursor.c [hunk 24]
```diff
@@ -83,7 +81,8 @@ pysqlite_cursor_init_impl(pysqlite_Cursor *self,
     return 0;
 }
 
-static void pysqlite_cursor_dealloc(pysqlite_Cursor* self)
+static void
```
Comment (berkerpeksag) [inline Modules/_sqlite/cursor.c]: Same comment about unrelated cosmetic changes.

File: Modules/_sqlite/cursor.c [hunk 25]
```diff
@@ -133,7 +132,7 @@ _pysqlite_get_converter(const char *keystr, Py_ssize_t keylen)
 }
 
 static int
-pysqlite_build_row_cast_map(pysqlite_Cursor* self)
+pysqlite_build_row_cast_map(pysqlite_Cursor *self)
```
Comment (berkerpeksag) [inline Modules/_sqlite/cursor.c]: Same comment about unrelated cosmetic changes.

File: Modules/_sqlite/cursor.c [hunk 26]
```diff
@@ -240,7 +239,7 @@ _pysqlite_build_column_name(pysqlite_Cursor *self, const char *colname)
  * - sqlite3_step() has been called before and it returned SQLITE_ROW.
  */
 static PyObject *
-_pysqlite_fetch_one_row(pysqlite_Cursor* self)
+_pysqlite_fetch_one_row(pysqlite_Cursor *self)
```
Comment (berkerpeksag) [inline Modules/_sqlite/cursor.c]: Same comment about unrelated cosmetic changes.

File: Modules/_sqlite/cursor.c [hunk 27]
```diff
@@ -366,7 +365,8 @@ _pysqlite_fetch_one_row(pysqlite_Cursor* self)
  *
  * 0 => error; 1 => ok
  */
-static int check_cursor(pysqlite_Cursor* cur)
+static int
```
Comment (berkerpeksag) [inline Modules/_sqlite/cursor.c]: Same comment about unrelated cosmetic changes.

File: Modules/_sqlite/cursor.c [hunk 28]
```diff
@@ -387,7 +387,8 @@ static int check_cursor(pysqlite_Cursor* cur)
 }
 
 static PyObject *
-_pysqlite_query_execute(pysqlite_Cursor* self, int multiple, PyObject* operation, PyObject* second_argument)
+_pysqlite_query_execute(pysqlite_Cursor *self, int multiple,
```
Comment (berkerpeksag) [inline Modules/_sqlite/cursor.c]: Same comment about unrelated cosmetic changes.

File: Modules/_sqlite/cursor.c [hunk 29]
```diff
@@ -1018,7 +1020,8 @@ static PyType_Spec cursor_spec = {
 
 PyTypeObject *pysqlite_CursorType = NULL;
 
-extern int pysqlite_cursor_setup_types(PyObject *module)
+int
```
Comment (berkerpeksag) [inline Modules/_sqlite/cursor.c]: This can be done in a separate PR.

File: Modules/_sqlite/module.c [hunk 30]
```diff
@@ -56,8 +56,8 @@ PyObject* _pysqlite_converters = NULL;
 int _pysqlite_enable_callback_tracebacks = 0;
 int pysqlite_BaseTypeAdapted = 0;
 
-static PyObject* module_connect(PyObject* self, PyObject* args, PyObject*
-        kwargs)
+static PyObject *
```
Comment (berkerpeksag) [inline Modules/_sqlite/module.c]: Same comment about unrelated cosmetic changes.

File: Modules/_sqlite/module.c [hunk 31]
```diff
@@ -256,7 +256,8 @@ pysqlite_adapt_impl(PyObject *module, PyObject *obj, PyObject *proto,
     return pysqlite_microprotocols_adapt(obj, proto, alt);
 }
 
-static int converters_init(PyObject* module)
+static int
```
Comment (berkerpeksag) [inline Modules/_sqlite/module.c]: Same comment about unrelated cosmetic changes.

File: Modules/_sqlite/module.c [hunk 32]
```diff
@@ -281,7 +282,8 @@ static PyMethodDef module_methods[] = {
     {NULL, NULL}
 };
 
-static int add_integer_constants(PyObject *module) {
+static int
```
Comment (berkerpeksag) [inline Modules/_sqlite/module.c]: Same comment about unrelated cosmetic changes.

File: Modules/_sqlite/module.c [hunk 33]
```diff
@@ -361,7 +363,8 @@ do {                                                            \
     }                                                           \
 } while (0)
 
-PyMODINIT_FUNC PyInit__sqlite3(void)
+PyMODINIT_FUNC
```
Comment (berkerpeksag) [inline Modules/_sqlite/module.c]: Same comment about unrelated cosmetic changes.

File: Modules/_sqlite/prepare_protocol.c [hunk 34]
```diff
@@ -52,7 +55,8 @@ static PyType_Spec type_spec = {
 
 PyTypeObject *pysqlite_PrepareProtocolType = NULL;
 
-extern int pysqlite_prepare_protocol_setup_types(PyObject *module)
+int
```
Comment (berkerpeksag) [inline Modules/_sqlite/prepare_protocol.c]: This can be done in a separate PR.

File: Modules/_sqlite/row.c [hunk 35]
```diff
@@ -150,7 +152,7 @@ PyObject* pysqlite_row_subscript(pysqlite_Row* self, PyObject* idx)
 }
 
 static Py_ssize_t
-pysqlite_row_length(pysqlite_Row* self)
+pysqlite_row_length(pysqlite_Row *self)
```
Comment (berkerpeksag) [inline Modules/_sqlite/row.c]: Same comment about unrelated cosmetic changes.

File: Modules/_sqlite/row.c [hunk 36]
```diff
@@ -184,17 +186,20 @@ pysqlite_row_keys_impl(pysqlite_Row *self)
     return list;
 }
 
-static PyObject* pysqlite_iter(pysqlite_Row* self)
+static PyObject *
```
Comment (berkerpeksag) [inline Modules/_sqlite/row.c]: Same comment about unrelated cosmetic changes.

File: Modules/_sqlite/row.c [hunk 37]
```diff
@@ -184,17 +186,20 @@ pysqlite_row_keys_impl(pysqlite_Row *self)
...
+pysqlite_iter(pysqlite_Row *self)
 {
     return PyObject_GetIter(self->data);
 }
 
-static Py_hash_t pysqlite_row_hash(pysqlite_Row *self)
+static Py_hash_t
```
Comment (berkerpeksag) [inline Modules/_sqlite/row.c]: Same comment about unrelated cosmetic changes.

File: Modules/_sqlite/row.c [hunk 38]
```diff
@@ -184,17 +186,20 @@ pysqlite_row_keys_impl(pysqlite_Row *self)
...
+pysqlite_row_hash(pysqlite_Row *self)
 {
     return PyObject_Hash(self->description) ^ PyObject_Hash(self->data);
 }
 
-static PyObject* pysqlite_row_richcompare(pysqlite_Row *self, PyObject *_other, int opid)
+static PyObject *
```
Comment (berkerpeksag) [inline Modules/_sqlite/row.c]: Same comment about unrelated cosmetic changes.

File: Modules/_sqlite/row.c [hunk 39]
```diff
@@ -241,7 +246,8 @@ static PyType_Spec row_spec = {
 
 PyTypeObject *pysqlite_RowType = NULL;
 
-extern int pysqlite_row_setup_types(PyObject *module)
+int
```
Comment (berkerpeksag) [inline Modules/_sqlite/row.c]: This can be done in a separate PR.

File: Modules/_sqlite/statement.c [hunk 40]
```diff
@@ -48,7 +48,9 @@ typedef enum {
     TYPE_UNKNOWN
 } parameter_type;
 
-int pysqlite_statement_create(pysqlite_Statement* self, pysqlite_Connection* connection, PyObject* sql)
+int
```
Comment (berkerpeksag) [inline Modules/_sqlite/statement.c]: Same comment about unrelated cosmetic changes.

File: Modules/_sqlite/statement.c [hunk 41]
```diff
@@ -112,7 +114,9 @@ int pysqlite_statement_create(pysqlite_Statement* self, pysqlite_Connection* con
     return rc;
 }
 
-int pysqlite_statement_bind_parameter(pysqlite_Statement* self, int pos, PyObject* parameter)
+int
```
Comment (berkerpeksag) [inline Modules/_sqlite/statement.c]: Same comment about unrelated cosmetic changes.

File: Modules/_sqlite/statement.c [hunk 42]
```diff
@@ -190,7 +194,8 @@ int pysqlite_statement_bind_parameter(pysqlite_Statement* self, int pos, PyObjec
 }
 
 /* returns 0 if the object is one of Python's internal ones that don't need to be adapted */
-static int _need_adapt(PyObject* obj)
+static int
```
Comment (berkerpeksag) [inline Modules/_sqlite/statement.c]: Same comment about unrelated cosmetic changes.

File: Modules/_sqlite/statement.c [hunk 43]
```diff
@@ -204,7 +209,9 @@ static int _need_adapt(PyObject* obj)
     }
 }
 
-void pysqlite_statement_bind_parameters(pysqlite_Statement* self, PyObject* parameters)
+void
```
Comment (berkerpeksag) [inline Modules/_sqlite/statement.c]: Same comment about unrelated cosmetic changes.

File: Modules/_sqlite/statement.c [hunk 44]
```diff
@@ -327,7 +334,8 @@ void pysqlite_statement_bind_parameters(pysqlite_Statement* self, PyObject* para
     }
 }
 
-int pysqlite_statement_finalize(pysqlite_Statement* self)
+int
```
Comment (berkerpeksag) [inline Modules/_sqlite/statement.c]: Same comment about unrelated cosmetic changes.

File: Modules/_sqlite/statement.c [hunk 45]
```diff
@@ -344,7 +352,8 @@ int pysqlite_statement_finalize(pysqlite_Statement* self)
     return rc;
 }
 
-int pysqlite_statement_reset(pysqlite_Statement* self)
+int
```
Comment (berkerpeksag) [inline Modules/_sqlite/statement.c]: Same comment about unrelated cosmetic changes.

File: Modules/_sqlite/statement.c [hunk 46]
```diff
@@ -363,12 +372,14 @@ int pysqlite_statement_reset(pysqlite_Statement* self)
     return rc;
 }
 
-void pysqlite_statement_mark_dirty(pysqlite_Statement* self)
+void
```
Comment (berkerpeksag) [inline Modules/_sqlite/statement.c]: Same comment about unrelated cosmetic changes.

File: Modules/_sqlite/statement.c [hunk 47]
```diff
@@ -398,7 +409,8 @@ void pysqlite_statement_dealloc(pysqlite_Statement* self)
  *
  * Returns 1 if there is more left than should be. 0 if ok.
  */
-static int pysqlite_check_remaining_sql(const char* tail)
+static int
```
Comment (berkerpeksag) [inline Modules/_sqlite/statement.c]: Same comment about unrelated cosmetic changes.

File: Modules/_sqlite/statement.c [hunk 48]
```diff
@@ -479,7 +491,8 @@ static PyType_Spec stmt_spec = {
 };
 PyTypeObject *pysqlite_StatementType = NULL;
 
-extern int pysqlite_statement_setup_types(PyObject *module)
+int
```
Comment (berkerpeksag) [inline Modules/_sqlite/statement.c]: This can be done in a separate PR.

File: Modules/_sqlite/util.c [hunk 49]
```diff
@@ -24,7 +24,8 @@
 #include "module.h"
 #include "connection.h"
 
-int pysqlite_step(sqlite3_stmt* statement, pysqlite_Connection* connection)
+int
+pysqlite_step(sqlite3_stmt *statement, pysqlite_Connection *connection)
```
Comment (berkerpeksag) [inline Modules/_sqlite/util.c]: Same comment about unrelated cosmetic changes.

File: Modules/_sqlite/util.c [hunk 50]
```diff
@@ -45,7 +46,8 @@ int pysqlite_step(sqlite3_stmt* statement, pysqlite_Connection* connection)
  * Checks the SQLite error code and sets the appropriate DB-API exception.
  * Returns the error code (0 means no error occurred).
  */
-int _pysqlite_seterror(sqlite3* db, sqlite3_stmt* st)
+int
```
Comment (berkerpeksag) [inline Modules/_sqlite/util.c]: Same comment about unrelated cosmetic changes.

File: Modules/_sqlite/util.c [hunk 51]
```diff
@@ -104,7 +106,7 @@ int _pysqlite_seterror(sqlite3* db, sqlite3_stmt* st)
 #endif
 
 sqlite_int64
-_pysqlite_long_as_int64(PyObject * py_val)
+_pysqlite_long_as_int64(PyObject *py_val)
```
Comment (berkerpeksag) [inline Modules/_sqlite/util.c]: Same comment about unrelated cosmetic changes.

**Review by berkerpeksag — CHANGES_REQUESTED** [review]
This PR could be much shorter and easy to review without all of these noisy changes.

File: Modules/_sqlite/cache.c (hunk 2 above)
Comment (erlend-aasland) [reply Modules/_sqlite/cache.c]: > The scope of this PR is adding missing `static` keywords and it's OK to follow PEP 7 while adding them.

That's my impression from other PR's that has been reviewed by other core devs; embedding PEP 7 style changes seems to be the default, AFAICT.

> However, changes like these are out of scope of this PR.

Isn't that a contradiction of what you just said? :)

File: Modules/_sqlite/cache.c (hunk 3 above)
Comment (erlend-aasland) [reply Modules/_sqlite/cache.c]: I can remove the `extern` clean-up and submit a separate PR for those using the same issue number.

File: Modules/_sqlite/cache.c (hunk 2 above)
Comment (berkerpeksag) [reply Modules/_sqlite/cache.c]: > That's my impression from other PR's that has been reviewed by other core devs; embedding PEP 7 style changes seems to be the default, AFAICT.

That's true as long as you've already touched the same lines as part of the original task you were working on. It doesn't mean you can randomly apply style changes all over the place.

> Isn't that a contradiction of what you just said? :)

No. Most of the changes here are purely cosmetic (i.e. they already have the `static` keyword in-place) It's OK to follow PEP 7 while adding *missing* `static` keywords.

Comment (berkerpeksag) [reply Modules/_sqlite/cache.c]: To be clear, the following diff is OK:

```diff
- int foo(...)
+ static int
+ foo(...) 
{

}

void bar(...)
{

}

static void baz(...)
{

}
```

But this one is not OK:

```diff
- int foo(...)
+ static int
+ foo(...) 
{

}

- void bar(...)
+ void
+ bar(...)
{

}

- static void baz(...)
+ static void
+ baz(...)
{

}
```

**Comment by erlend-aasland** [comment]
I have made the requested changes; please review again

File: Modules/_sqlite/cache.c (hunk 2 above)
Comment (erlend-aasland) [reply Modules/_sqlite/cache.c]: BTW 1, Would you accept a PR that cleans up the remaining ~40 argument specs wrt. PEP 7? (Separate issue, of course)

BTW 2, neat with inline syntax highlighting in the GH markup. I like that. Thanks.

**Review by berkerpeksag — APPROVED** [review]
Much cleaner now, thank you!

**Comment by erlend-aasland** [comment]
> Much cleaner now, thank you!

Thanks, Berker. After thinking about it, I do agree with you.
